build.properties.default

```diff
@@ -1,4 +1,4 @@
 # Maven2 Repository Locations (you can override these in "build.properties" to point to a local proxy, e.g. Nexus)
 artifact.remoteRepository.central:     https://repo1.maven.org/maven2
 artifact.remoteRepository.apache:      https://repo.maven.apache.org/maven2
-
+artifact.remoteRepository.apacheSnapshots: https://repository.apache.org/content/groups/snapshots/
```

Comment:
spoke in slack, please remove; I am fine if on-commit you drop but we should avoid snapshots

Reply:
removed